Description: accept newer minor version of crate heck
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2026-04-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/crates/wasm-compose/Cargo.toml
+++ b/crates/wasm-compose/Cargo.toml
@@ -16,7 +16,7 @@
 
 [dependencies]
 anyhow = { workspace = true }
-heck = "0.4.0"
+heck = ">= 0.4.0, <= 0.5.0"
 im-rc = "15.1.0"
 indexmap = { workspace = true, features = ["serde"] }
 log = { workspace = true }
